src/bin/cargo/commands/install.rs

```diff
@@ -203,6 +203,7 @@ pub fn exec(config: &mut Config, args: &ArgMatches) -> CliResult {
             &compile_opts,
             args.flag("force"),
             args.flag("no-track"),
+            !args.flag("ignore-rust-version"),
```

Comment:
Is it possible to access `compile_opts.honor_rust_version` instead of adding another parameter?